config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-gcr-to-ar-sync.yaml

```diff
@@ -1,33 +0,0 @@
-# If you are adding new AR regions, you need to run this job for a while and then add the manifests in k/k8s.io
-periodics:
-  - name: gcr-to-ar-sync-us
-    cron: "0 */2 * * *" # 2h
-    # cron: "0 0 31 4 *" # 31st of April
```

Comment:
I would probably just use this cron instead. This job is needed when new repos are added.